### DISCUSSION TITLE
There is a method to add a class to a swiper-slide in angular ?

### DISCUSSION THREAD

**Discussion by gvastethecreator** [nolimits4web/swiper#5561] [Q&A] [answered]
Hi, i want to know if it's possible to add a class to the slides, like using ngClass, but that's not possible with ng-template so i want to known if exists a workaround for this, i have a slide that when clicked add an "active" class but i need to be added to the swiper-slide instead of the content inside.

**Comment by vltansky** [comment, marked as answer]
https://github.com/nolimits4web/swiper/blob/master/src/angular/src/swiper-slide.directive.ts#L8

You can use `class`. See https://codesandbox.io/s/swiper-default-angular-forked-e3zlln?file=/src/app/app.component.ts
`ngClass` support (https://github.com/nolimits4web/swiper/pull/5562) will be released soon.
